src/components/calcite-input/calcite-input.tsx

```diff
@@ -195,10 +195,15 @@ export class CalciteInput implements LabelableComponent, FormComponent {
     | "week" = "text";
 
   /** input value */
-  @Prop({ mutable: true }) value: string;
+  @Prop({ mutable: true }) value = "";
 
   @Watch("value")
   valueWatcher(newValue: string): void {
+    if (newValue == null) {
+      this.value = "";
+      return;
+    }
+
```

Comment:
Does this statement need to handle `undefined` as well?

Reply:
It already handles it since this isn't using strict equality.